src-docs/src/views/facet/playground.js

```diff
@@ -0,0 +1,103 @@
+import { PropTypes } from 'react-view';
+import {
+  EuiFacetButton,
+  EuiIcon,
+  EuiAvatar,
+  EuiFacetGroup,
+} from '../../../../src/components/';
+import { propUtilityForPlayground } from '../../services/playground';
+import * as t from '@babel/types';
+
+export const facetButtonConfig = () => {
+  const docgenInfo = Array.isArray(EuiFacetButton.__docgenInfo)
+    ? EuiFacetButton.__docgenInfo[0]
+    : EuiFacetButton.__docgenInfo;
+  const propsToUse = propUtilityForPlayground(docgenInfo.props);
+
+  propsToUse.onClick = {
+    ...propsToUse.onClick,
+    type: PropTypes.Custom,
+    value: undefined,
+    custom: {
+      ...propsToUse.onClick.custom,
+      use: 'switch',
+      label: 'Simulate',
+    },
+  };
+  propsToUse.children = {
+    type: PropTypes.String,
+    value: 'facet content',
```

Comment:
Place be mindful to use sentence case always
```suggestion
    value: 'Facet content',
```